### ISSUE TITLE
Support GPU on M3

### ISSUE THREAD

**Issue by narugit** [narugit/smctemp#26]

**Comment by narugit** [comment]
About this Mac→Detail→System Report→Graphics/Display
![image](https://github.com/narugit/smctemp/assets/28133383/bab4494c-5757-4976-b5cf-1b9fcfa9e5ae)
10 Core

**Comment by narugit** [comment]
```console
$ ❯ system_profiler SPDisplaysDataType

Graphics/Displays:

    Apple M3:

      Chipset Model: Apple M3
      Type: GPU
      Bus: Built-In
      Total Number of Cores: 10
      Vendor: Apple (0x106b)
      Metal Support: Metal 3
      Displays:
        Color LCD:
          Display Type: Built-in Liquid Retina XDR Display
          Resolution: 3024 x 1964 Retina
          Main Display: Yes
          Mirror: Off
          Online: Yes
          Automatically Adjust Brightness: No
          Connection Type: Internal
        ILD-A27FHD-B:
          Resolution: 1920 x 1080 (1080p FHD - Full High Definition)
          UI Looks like: 1920 x 1080 @ 60.00Hz
          Mirror: Off
          Online: Yes
          Rotation: Supported
```

For CPU: SPHardwareDataType

**Comment by narugit** [comment]
```console
$ smctemp -l | grep ^"  Tg"
  Tg0C  [flt ]  38.3 (bytes: 00 2C 19 42)
  Tg0D  [flt ]  44.2 (bytes: 9A C5 30 42)
  Tg0O  [flt ]  38.5 (bytes: 22 08 1A 42)
  Tg0P  [flt ]  44.4 (bytes: BC A1 31 42)
  Tg0W  [flt ]  38.3 (bytes: CE 3B 19 42)
  Tg0X  [flt ]  44.2 (bytes: 68 D5 30 42)
  Tg0a  [flt ]  38.1 (bytes: 43 90 18 42)
  Tg0b  [flt ]  44.0 (bytes: DD 29 30 42)
  Tg0i  [flt ]  38.1 (bytes: 58 67 18 42)
  Tg0j  [flt ]  44.0 (bytes: F2 00 30 42)
  Tg0u  [flt ]  38.4 (bytes: B0 8E 19 42)
  Tg0v  [flt ]  44.3 (bytes: 4A 28 31 42)

$ smctemp -l | grep ^"  Tg" | wc -l
      12
```

**Comment by narugit** [comment]
No gpu sensors start from `Tf` in my M3
https://github.com/exelban/stats/blob/c37585d4b3731acadce5dde3b77b69b12a63cb18/Modules/Sensors/values.swift#L410-L417
```console
$ smctemp -l | grep ^"  Tf"
```

**Comment by narugit** [comment]
Fix by #27
